oggsound/src/OgreOggISound.cpp

```diff
@@ -723,8 +723,11 @@ namespace OgreOggSound
 			#endif
 		);
 
+		#if OGRE_VERSION_MAJOR == 1
```

Comment:
```suggestion
		#if OGRE_VERSION_MAJOR != 2
```
Ogre 13 is OGRE_VERSION_MAJOR=13

Reply:
fixed